site/cypress/integration/courseMaterials.spec.js

```diff
@@ -356,5 +356,6 @@ describe('Test cases revolving around course material uploading and access contr
 
         cy.get('.fa-trash').first().click();
         cy.get('.btn-danger').click();
+        cy.wait(1000);
```

Comment:
What does the wait here at the end of the test accomplish? All prior commands have executed at this point? Or is this just to protect on the teardown method?

Reply:
Yeah it looks like there was an issue with loading the page after being finished and running the logout/login after each test.
The page not being response or cypress not seeing that its loaded, but it doesn't appear to fix it completely